src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -223,7 +223,7 @@ ObjectField.propTypes = {
     hideHeaderRow: PropTypes.bool,
     hideTitle: PropTypes.bool,
     onEdit: PropTypes.func,
-    pageTitle: PropTypes.string,
+    pageTitle: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
```

Comment:
This resolves a console error that was cropping up with the page title functions.